bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZ_leptonic/WLLjj_dim8_ewk_run_card.dat

```diff
@@ -0,0 +1,269 @@
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0  = ej     ! minimum E for the jets 
+  0  = eb     ! minimum E for the b 
+  0  = ea     ! minimum E for the photons 
+  0  = el     ! minimum E for the charged leptons 
+ -1   = ejmax ! maximum E for the jets
+ -1   = ebmax ! maximum E for the b
+ -1   = eamax ! maximum E for the photons
+ -1   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  -1  = etaj    ! max rap for the jets 
+  -1  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons 
+  -1  = etal    ! max rap for the charged leptons 
+ 0.6  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+ 0.0 = drjj    ! min distance between jets 
+ 0.0 = drbb    ! min distance between b's 
+ 0.01 = drll    ! min distance between leptons 
+ 0.0 = draa    ! min distance between gammas 
+ 0.0 = drbj    ! min distance between b and jet 
+ 0.0 = draj    ! min distance between gamma and jet 
+ 0.0 = drjl    ! min distance between jet and lepton 
+ 0.0 = drab    ! min distance between gamma and b 
+ 0.0 = drbl    ! min distance between b and lepton 
+ 0.0 = dral    ! min distance between gamma and lepton 
+ -1  = drjjmax ! max distance between jets
+ -1  = drbbmax ! max distance between b's
+ -1  = drllmax ! max distance between leptons
+ -1  = draamax ! max distance between gammas
+ -1  = drbjmax ! max distance between b and jet
+ -1  = drajmax ! max distance between gamma and jet
+ -1  = drjlmax ! max distance between jet and lepton
+ -1  = drabmax ! max distance between gamma and b
+ -1  = drblmax ! max distance between b and lepton
+ -1  = dralmax ! maxdistance between gamma and lepton
+#*********************************************************************
+# Minimum and maximum invariant mass for pairs                       *
+# WARNING: for four lepton final state mmll cut require to have      *
+#          different lepton masses for each flavor!                  *           
+#*********************************************************************
+ 100 = mmjj    ! min invariant mass of a jet pair 
+ 100 = mmbb    ! min invariant mass of a b pair 
+ 0   = mmaa    ! min invariant mass of gamma gamma pair
+ 60  = mmll    ! min invariant mass of l+l- (same flavour) lepton pair
```

Comment:
"60 = mmll" seems to be a very strong cut at ME level, since you decay W/Z in the syntax, the two leptons in mmll could come from W or Z. i just checked previous cards for WZ https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WLLJJ_WToLNu_aQGC-FM_EWKOnly/highMll, you can decide whether you want to use similar syntax (using madspin card) or set "mmll" to a looser value

Reply:
Okay, I have softened the cut to 40.

Reply:
Hi, when i perform some checks on the mmll cut, i find current syntax would be incompatible with mmll cut according to https://answers.launchpad.net/mg5amcnlo/+question/273813, so maybe you can use syntax "generate  p p > w+ l+ l- j j QED=5 QCD=0", and then use mmll=40 to select the Z events, and also set "False=cut_decays" to avoid such incompatibility as mentioned.

Reply:
Hi, @menglu21 These cards are for the signal sample and we want W & Z as intermediate particles.

Reply:
Hi @geet59, i mean with current syntax you will get error because you use multiple particle "l+" in the syntax and also require mmll and this will cause error, if you want to use current syntax (you can have a test with the local MG), you man need to set 'mmll=0.0', but this will make many of the events not from the Z resonance. my last suggestion is that you can "p p > w+ l l j j", and require, e.g., "mmll=50" to make most of the contribution comes from Z events.

Reply:
Hi, @menglu21 So you suggest modifying like this?

generate    p p > w+ l l j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w+ > l+ vl @ 1
add process p p > w- l l j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w- > l- vl~ @ 2

and mmll=50 in the run card?


or 
generate    p p > w+ l+ l- j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w+ > l+ vl @ 1
add process p p > w- l+ l- j j QED=4 QCD=0 S0=1 S1=1 S2=1 M0=1 M1=1 M2=1 M3=1 M4=1 M5=1 M6=1 M7=1 T0=1 T1=1 T2=1 T5=1 T6=1 T7=1, w- > l- vl~ @ 2

I am basically confused with "p p > w+ l l j j" l here. Please clarify?

Reply:
Hi, "l" in "p p > w+ l l j j" is a typo, "l+" is correct. and please don't forget use "False=cut_decays"

Reply:
Hi, I have made the changes. I hope it is okay now.

Reply:
want to make sure "mmjj=100" is you want, e.g., use low mjj region as a control region check

Reply:
Hi, this is our signal sample and we require mmjj>500.

Reply:
yeah, so i mean if you just need mmjj>500, then you can tighten the cut in run card, currently you use 'mmjj=100'

Reply:
Sorry, for CR we have a requirement of mmjj>200 and for SR mmjj>500. So I think it mmjj>100 is also fine or should I make mmjj>200?

Reply:
i see, then i think current setup is fine. @Saptaparna @GurpreetSinghChahal @mlizzo please have a check

Reply:
Hi @menglu21 @Saptaparna @GurpreetSinghChahal @mlizzo any updates on this?

Reply:
My apologies for the delay. Just followed the discussion and agree with Meng.